clevertap-core/src/main/java/com/clevertap/android/sdk/Utils.java

```diff
@@ -564,4 +525,43 @@ public static boolean isMainProcess(Context context, String mainProcessName) {
         }
         return false;
     }
+
+    public static List<JSONObject> toJSONObjectList(JSONArray jsonArray) throws JSONException {
```

Comment:
why do we need to create this list in first place when we can directly iterate on JSONArray?

Reply:
@shivamsharma2710